### PR TITLE
Updated README.md for the next tag

### PR DESCRIPTION
**Pull request by smalti**

### CODE REVIEW

File: src/gmock-win32.cpp [hunk 1]
```diff
@@ -232,13 +232,13 @@ namespace utils {
                 rvaToVa< IMAGE_THUNK_DATA >(base, descr->OriginalFirstThunk) : thunkIAT;
 
             for (; thunk->u1.Function; ++thunk, ++thunkIAT)
-        {
-                if (!descr->OriginalFirstThunk || thunk->u1.Ordinal & IMAGE_ORDINAL_FLAG)
             {
-                    if (*thunkProc(thunkIAT) == funcAddr)
+                if (!descr->OriginalFirstThunk || thunk->u1.Ordinal & IMAGE_ORDINAL_FLAG)
```
Comment (smalti) [inline src/gmock-win32.cpp:236]: Unrelated change: fixed alignments

File: README.md [hunk 2]
```diff
@@ -64,7 +64,9 @@ ON_MODULE_FUNC_CALL(GetCurrentProcessId).WillByDefault(Return(42));
 ON_MODULE_FUNC_CALL(GetProcessIdOfThread, Eq(HANDLE(42))).WillByDefault(Return(1));
 ```
 
-#### 4. To validate and clear expectations after tests use the `VERIFY_AND_CLEAR_MODULE_FUNC` or `VERIFY_AND_CLEAR_MODULE_FUNC_EXPECTATIONS` macro (usually used for multiple tests with different expectations and can be executed as part of the test tear-down logic)
+#### 4. To validate and clear expectations after tests use the `VERIFY_AND_CLEAR_MODULE_FUNC` or `VERIFY_AND_CLEAR_MODULE_FUNC_EXPECTATIONS` macro
+
+In `GMock` the expectations of a mock object are verified upon its destruction. However, since we do not destroy our objects, we must manually verify our expectations after test. In the general case, we should verify expectations after every test case, similar to how a Mock object would be destroyed. This can typically be done in the `TearDown` method:
```
Comment (Alex0vSky) [inline README.md:70]: Important note, good to have it here.

**Review by Alex0vSky — APPROVED** [review]
